src/options.cpp

```diff
@@ -1569,106 +1583,121 @@ void options_manager::add_options_general()
-    "ask"
-       );
+        add( "DEATHCAM", page_id, to_translation( "DeathCam" ),
+             to_translation( "Always: Always start deathcam.  Ask: Query upon death.  Never: Never show deathcam." ),
+        { { "always", to_translation( "Always" ) }, { "ask", to_translation( "Ask" ) }, { "never", to_translation( "Never" ) } },
+        "ask"
+           );
 
-    add( "EVENT_SPAWNS", "general", to_translation( "Special event spawns" ),
+    add( "EVENT_SPAWNS", page_id, to_translation( "Special event spawns" ),
          to_translation( "If not disabled, unique items and/or monsters can spawn during special events (Christmas, Halloween, etc.)" ),
     { { "off", to_translation( "Disabled" ) }, { "items", to_translation( "Items" ) }, { "monsters", to_translation( "Monsters" ) }, { "both", to_translation( "Both" ) } },
     "off" );
+    });
 
     add_empty_line();
 
-    add( "SOUND_ENABLED", "general", to_translation( "Sound enabled" ),
+    add_option_group( "general", Group( "soundpacks_opts", to_translation( "Soundpack Options" ),
+                                          to_translation( "Options regarding Soundpack." ) ),
+    [&]( const std::string & page_id ) {    [&]( const std::string & page_id ) {
+    add( "SOUND_ENABLED", page_id, to_translation( "Sound enabled" ),
          to_translation( "If true, music and sound are enabled." ),
          true, COPT_NO_SOUND_HIDE
        );
 
-    add( "SOUNDPACKS", "general", to_translation( "Choose soundpack" ),
+    add( "SOUNDPACKS", page_id, to_translation( "Choose soundpack" ),
          to_translation( "Choose the soundpack you want to use.  Requires restart." ),
          build_soundpacks_list(), "basic", COPT_NO_SOUND_HIDE
        ); // populate the options dynamically
 
     get_option( "SOUNDPACKS" ).setPrerequisite( "SOUND_ENABLED" );
 
-    add( "MUSIC_VOLUME", "general", to_translation( "Music volume" ),
+    add( "MUSIC_VOLUME", page_id, to_translation( "Music volume" ),
          to_translation( "Adjust the volume of the music being played in the background." ),
          0, 128, 100, COPT_NO_SOUND_HIDE
        );
 
     get_option( "MUSIC_VOLUME" ).setPrerequisite( "SOUND_ENABLED" );
 
-    add( "SOUND_EFFECT_VOLUME", "general", to_translation( "Sound effect volume" ),
+    add( "SOUND_EFFECT_VOLUME", page_id, to_translation( "Sound effect volume" ),
          to_translation( "Adjust the volume of sound effects being played by the game." ),
          0, 128, 100, COPT_NO_SOUND_HIDE
        );
 
     get_option( "SOUND_EFFECT_VOLUME" ).setPrerequisite( "SOUND_ENABLED" );
 
-    add( "AMBIENT_SOUND_VOLUME", "general", to_translation( "Ambient sound volume" ),
+    add( "AMBIENT_SOUND_VOLUME", page_id, to_translation( "Ambient sound volume" ),
          to_translation( "Adjust the volume of ambient sounds being played by the game." ),
          0, 128, 100, COPT_NO_SOUND_HIDE
        );
```

Comment:
The previous opening brackets at line 1671-1673 are unclosed but I cannot add the suggestion below line 1705.